server/src/main/java/io/littlehorse/server/KafkaStreamsServerImpl.java

```diff
@@ -241,7 +241,7 @@ private RequestExecutionContext requestContext() {
     public KafkaStreamsServerImpl(LHServerConfig config) {
         MetadataCache metadataCache = new MetadataCache();
         this.config = config;
-        this.taskQueueManager = new TaskQueueManager(this);
+        this.taskQueueManager = new TaskQueueManager(this, 1_000);
```

Comment:
Can we move this to `LHConstants.MAX_TASKRUNS_IN_ONE_TASKQUEUE` or something like that?